concent_api/core/models.py

```diff
@@ -684,3 +685,50 @@ class PendingEthereumTransaction(Model):
     s = DecimalField(max_digits=78, decimal_places=0)
 
     created_at = DateTimeField(auto_now_add=True)
+
+
+class DepositAccount(Model):
+    client = ForeignKey(Client)
+    ethereum_public_key = CharField(max_length=ETHEREUM_ADDRESS_LENGTH)
+    created_at = DateTimeField(auto_now_add=True)
+
+    def clean(self) -> None:
+        super().clean()
+        if not isinstance(self.ethereum_public_key, str) or len(self.ethereum_public_key) != 42:
+            raise ValidationError({
+                'ethereum_public_key': 'The length of ethereum_public_key must be exactly 42 characters.'
+            })
+
+
+class DepositClaim(Model):
+    subtask = ForeignKey(Subtask, blank=True, null=True)
+    payer_deposit_account = ForeignKey(DepositAccount)
+    payee_ethereum_public_key = CharField(max_length=ETHEREUM_ADDRESS_LENGTH)
+    concent_use_case = IntegerField()
+    amount = DecimalField(max_digits=78, decimal_places=0)
+    tx_hash = CharField(max_length=64, blank=True, null=True, unique=True)
+    created_at = DateTimeField(auto_now_add=True)
+    modified_at = DateTimeField(auto_now=True)
+
+    def clean(self) -> None:
+        super().clean()
+        if self.subtask is None and self.concent_use_case != ConcentUseCase.FORCED_PAYMENT:
+            raise ValidationError({
+                'subtask: Can be NULL if and only if concent_use_case is ForcedPayment'
+            })
+        if not isinstance(self.payer_deposit_account, DepositAccount) or self.payer_deposit_account.ethereum_public_key == self.payee_ethereum_public_key:
+            raise ValidationError({
+                'payer_deposit_account': 'payer_deposit_account.ethereum_public_key cannot be the same as payee_ethereum_public_key'
+            })
+        if not isinstance(self.payee_ethereum_public_key, str) or self.payee_ethereum_public_key == self.payer_deposit_account.ethereum_public_key:
```

Comment:
Please add validation for `tx_hash` if it is set.

Reply:
Done.

Reply:
Should be:
```
if not (
    self.tx_hash is None or 
    isinstance(self.tx_hash, str)
) or len(self.tx_hash) != 64:
```

Please make that `64` a constant and use where applicable.